hw/top_darjeeling/sw/autogen/top_darjeeling_soc_mbx.h

```diff
@@ -0,0 +1,211 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SOC_MBX_HW_TOP_DARJEELING_SW_AUTOGEN_TOP_DARJEELING_H_
+#define OPENTITAN_SOC_MBX_HW_TOP_DARJEELING_SW_AUTOGEN_TOP_DARJEELING_H_
+
+#include "top_darjeeling_flash_ctrl_dummy.h"
```

Comment:
That might be something to remove from the generation.

Reply:
And the order of the top and address apce might be changed in the include guard.